changhyun/app.js

```diff
@@ -3,8 +3,9 @@ require("dotenv").config();
 const express = require("express");
 const cors = require("cors");
 const morgan = require("morgan");
-const { DataSource } = require("typeorm");
+const { DataSource, SimpleConsoleLogger } = require("typeorm");
 const app = express();
+const bcrypt = require("bcrypt");
 
```

Comment:
```suggestion
const { DataSource, SimpleConsoleLogger } = require("typeorm");
const bcrypt = require("bcrypt");

const app = express();
```
module import convention 수정입니다!